RoddyCore/src/de/dkfz/roddy/ExitReasons.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016 German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ).
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/TheRoddyWMS/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy
+
+import groovy.transform.CompileStatic
+
+@CompileStatic
+enum ExitReasons {
+
+    wrongStartupLocation(255, "You must call roddy from the right location! (Base roddy folder where roddy.sh resides."),
```

Comment:
Please try to make the messages as concise as possible.

"Wrong execution directory. Change to directory with roddy.sh before execution."